File: main/contrib/Windows API Code Pack 1.1/source/Samples/DirectX/VB/Direct2D/RandomShapes/Window.vb
' Copyright (c) Microsoft Corporation.  All rights reserved.

Imports Microsoft.VisualBasic
Imports System

Imports System.Windows.Forms

Namespace RandomShapes
	Partial Public Class Window
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Window_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			d2DShapesControlWithButtons1.Initialize()
		End Sub
	End Class
End Namespace

File: main/contrib/Windows API Code Pack 1.1/source/Samples/DirectX/CS/Direct2D/RandomShapes/Window.cs
// Copyright (c) Microsoft Corporation.  All rights reserved.

using System;

using System.Windows.Forms;

namespace RandomShapes
{
    public partial class Window : Form
    {
        public Window()
        {
            InitializeComponent();
        }

        private void Window_Load(object sender, EventArgs e)
        {
            d2DShapesControlWithButtons1.Initialize();
        }
    }
}
